tests/contrib/flask/web.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+import os
+
+from flask import Flask, render_template
+
+
+class TestError(Exception):
+    pass
+
+
+class HandleMe(Exception):
+    pass
+
+
+def create_app():
+    """Initializes a new Flask application. This method is required to
+    be sure each time a test is executed, the Flask app is always new
+    and without any tracing side effect from the previous execution.
+    """
+    cur_dir = os.path.dirname(os.path.realpath(__file__))
+    tmpl_path = os.path.join(cur_dir, 'test_templates')
+    app = Flask(__name__, template_folder=tmpl_path)
+
+    @app.route('/')
+    def index():
+        return 'hello'
+
+    @app.route('/error')
+    def error():
+        raise TestError()
+
+    @app.route('/handleme')
+    def handle_me():
+        raise HandleMe()
+
+    @app.route('/fatal')
```

Comment:
moving this route handler to this file breaks this line of this test: https://github.com/DataDog/dd-trace-py/blob/23287fea06961faf06c8a695fabe01b83e56b68e/tests/contrib/flask/test_flask.py#L274

since now the error is in `contrib/flask/web.py`

Reply:
nice catch! thank you very much!